src/batou_ext/cron.py

```diff
@@ -186,6 +187,11 @@ def configure(self):
                     User = "{{component.run_as}}";
                     ExecStart = "{{component.wrapped_command}}";
                     TimeoutStartSec = "{{component.timeout}}";
+                    {%- if component.additional_service_config is not none %}
```

Comment:
Ich bin mir nicht sicher, dass `none` statt `None` geht … bist du sicher?

Reply:
Ja, im Jinja ist das klein geschrieben und in meinem Test funktioniert es.